packages/interactivity/CHANGELOG.md

```diff
@@ -2,10 +2,15 @@
 
 ## Unreleased
 
+### Enhancements
+
+-   Merge new server-side rendered context on client-side navigation. ([#53853](https://github.com/WordPress/gutenberg/pull/53853))
+
 ## 2.1.0 (2023-08-16)
 
 ### New Features
 
+-   Support region-based client-side navigation. ([#53733](https://github.com/WordPress/gutenberg/pull/53733))
```

Comment:
For some reason, this has been moved to the 2.1.0 section when it should be in the Unreleased section.
```suggestion
### Enhancements

-   Merge new server-side rendered context on client-side navigation. ([#53853](https://github.com/WordPress/gutenberg/pull/53853))
-   Support region-based client-side navigation. ([#53733](https://github.com/WordPress/gutenberg/pull/53733))

## 2.1.0 (2023-08-16)

### New Features

```